### PR TITLE
feat: added faucet, vesting and evm withdraw UI

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* Added Faucet, EVM withdrawal, and Vesting UI
* Fixed modal(astar-ui library) issues (styling and light theme) by overwriting the CSS in [astar-ui.scss](https://github.com/AstarNetwork/astar-apps/blob/feat/v2-modals/src/css/astar-ui.scss) @sirius651 

**Check list**
- [ ] contains breaking changes
- [x] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes

**This pull request makes the following changes:**

=Before=
image

image

=After=
Transfer
image

image

EVM withdrawal
image

Vesting info
image

Faucet
image

image

- [ ] (ex: add user list)

### CODE REVIEW

File: src/components/assets/NativeAssetList.vue [hunk 1]
```diff
@@ -1,121 +1,173 @@
...
-          <span class="text--md">{{ $t('assets.haveDepositedFromEvm') }}</span>
-        </div>
-        <div class="row__right">
-          <div class="column--balance">
-            <div class="column__box">
-              <span class="text--value">{{ $n(numEvmDeposit) }} {{ tokenSymbol }}</span>
-            </div>
+        <div class="row--bg--extend row--details bg--accent">
+          <div class="row__left">
+            <span class="text--md">{{ $t('assets.haveDepositedFromEvm') }}</span>
```
Comment (hoonsubin) [inline src/components/assets/NativeAssetList.vue:68]: Screen Shot 2022-03-25 at 8 02 57 AM
I feel like the text is too long and it stands out. @Kahonnohak what do you think?

Comment (Kahonnohak) [reply src/components/assets/NativeAssetList.vue:68]: @hoonsubin 

"EVM deposit" is also fine, it is a little kinder to say ; you have the deposit from EVM account.

I'm not a mother ton English speaker so please correct the texts.
- EVM deposit
- You have the deposit from EVM account
- You have EVM deposit

Comment (Kahonnohak) [reply src/components/assets/NativeAssetList.vue:68]: One last change please! if we are to release Light theme at the same time could you change the dropshadow for the modal here to `0 0 15px 4px #00000030` 
スクリーンショット 2022-03-25 16 15 18

**Review by Kahonnohak — APPROVED** [review]
Have commented in the feed for a change thanks for amazing work!

**Comment by impelcrypto** [comment]
@sirius651  @Kahonnohak 
I've applied the light theme to some of the header items.
Commit: https://github.com/AstarNetwork/astar-apps/pull/294/commits/a647348b0560135cfc727d5a0316fcf40580fad5

image

File: src/components/assets/NativeAssetList.vue (hunk 1 above)
Comment (impelcrypto) [reply src/components/assets/NativeAssetList.vue:68]: fixed

Comment (impelcrypto) [reply src/components/assets/NativeAssetList.vue:68]: @hoonsubin How about `Your EVM deposit` ?

Comment (hoonsubin) [reply src/components/assets/NativeAssetList.vue:68]: Yep, that sounds good

Comment (impelcrypto) [reply src/components/assets/NativeAssetList.vue]: @hoonsubin 
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/294/commits/32f7513667288de968ae143bc8be857ff09f0c17) commit.
image

**Comment by sirius651** [comment]
> @sirius651 @Kahonnohak I've applied the light theme to some of the header items. Commit: [a647348](https://github.com/AstarNetwork/astar-apps/commit/a647348b0560135cfc727d5a0316fcf40580fad5)

Thanks!

File: src/components/assets/NativeAssetList.vue (hunk 1 above)
Comment (Kahonnohak) [reply src/components/assets/NativeAssetList.vue]: Thank you @impelcrypto !

**Review by sirius651 — APPROVED** [review]

**Comment by Kahonnohak** [comment]
@impelcrypto 
If you have time...

Light theme
- please check the line, make it 1 px please.
スクリーンショット 2022-03-26 23 49 31

@sirius651 
I'll check the header and the sidebar together and would ask some changes but let's say only if there is critical. The drawer modals are the priority.

**Comment by impelcrypto** [comment]
@Kahonnohak 

> Light theme
One mend please - the line is 1px.

Fixed.

**Review by hoonsubin — APPROVED** [review]
LGTM

**Comment by hoonsubin** [comment]
Please resolve the merge conflicts

**Pull request merged**